### PR TITLE
fix: added CorsOptions to email images

### LINKED ISSUE #6869: Images not showing on emails from Parabol
- **How serious is the bug?** Unsure (needs testing to see if a unique case or widerspread than one computer)
- **OS:** macOS
- **Email Client:** Apple Mail

### Acceptance Criteria
Images (or at least that seems to be the case) aren't showing on any emails from Parabol when see them from other services. The context is a new email account, using Apple Mail. Slack emails are showing images.

The expectation would be for images to show just like other services.

What is being seen from Parabol emails:

2022-07-11 at 22 10

What is being seen on the same email, same computer, same app (Apple Mail) for Loom (Slack, Notion and Figma also don't have issue):

2022-07-11 at 22 11

**Estimated effort:** 1 hour to triage to decide if unique case.

### PR DESCRIPTION
**Pull request by BartoszJarocki**
# Description

Fixes #6869 

## Demo
Screenshot 2022-11-04 at 16 20 36

## Testing scenarios

- [ ] Finish any meeting and see if email summary images are showing correctly in Apple Mail
- [ ] Make sure attribute `crossorigin=''` is present in the web app on all `` elements in the meeting summary

## Final checklist

- [x] I checked the [code review guidelines](../docs/codeReview.md)
- [x] I have added [Metrics Representative](../docs/codeReview.md#metrics-representative) as reviewer(s) if my PR invovles metrics/data/analytics related changes
- [x] I have performed a self-review of my code, the same way I'd do it for any other team member
- [x] I have tested all cases I listed in the testing scenarios and I haven't found any issues or regressions
- [x] Whenever I took a non-obvious choice I added a comment explaining why I did it this way
- [x] I added the label `One Review Required` if the PR introduces only minor changes, does not contain any architectural changes or does not introduce any new patterns and I think one review is sufficient'
- [x] PR title is human readable and could be used in changelog

### CODE REVIEW

File: packages/client/types/cors.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+ * More information here: https://stackoverflow.com/questions/39109789/what-limitations-apply-to-opaque-responses
+ */
+export const APP_CORS_OPTIONS: CorsOptions = {
+  crossOrigin: 'anonymous'
+}
+
+/**
+ * Using CORS in emails can cause issues with some email clients eg. images not showing in Apple Mail, let's disable it.
+ */
+export const EMAIL_CORS_OPTIONS: CorsOptions = {}
```
Comment (BartoszJarocki) [inline packages/client/types/cors.ts:20]: Some components are used in both emails and app. info in what context component is used is not always available. I decided that adding a separate prop with a cors option is the most viable solution right now, especially when we can have a set of predefined options for emails and app.

**Comment by BartoszJarocki** [comment]
@nickoferrall may I ask you to take a look? thanks!

**Review by nickoferrall — APPROVED** [review]
Tested and it LGTM!

**Comment by BartoszJarocki** [comment]
@Dschoordsch do you mind taking a look? thanks!

**Review by Dschoordsch — APPROVED** [review]
Looks good to me, but I did not test

**Pull request merged**